src/nsidc/metgen/models.py

```diff
@@ -0,0 +1,38 @@
+"""
+Data models for the metgen package.
+
+This module contains dataclasses used throughout the metgen pipeline.
+"""
+
+import dataclasses
+from typing import Optional
+
+
+@dataclasses.dataclass
+class CollectionMetadata:
+    """
+    Collection metadata relevant to generating UMM-G content.
+
+    This dataclass contains all the metadata fields needed for granule
+    metadata generation, retrieved from CMR or other sources.
+    """
+
+    # Collection identifiers
+    short_name: str  # Previously called auth_id
```

Comment:
I prefer the term auth_id because that's how we refer to it on the NSIDC side of the house, at least for the writers (and for me). (It is short for "authoritative ID.") If someone did a poll of operators and they prefer short_name in error messages because it's a direct map to CMR lingo, that makes some sense. Probably that makes more sense to you, too. 

Mainly this switching to `short_name` is a lot of cognitive dissonance for me because we use `auth_id` in all of our collection metadata management at NSIDC. But maybe I will just have to get over it.